src/data/full-quiz.ts

```diff
@@ -7152,5 +7152,27 @@ const fullQuiz = [
       "Npm scripts allow commandline tools to be called directly without relying on external plugins.",
     Link: "https://www.freecodecamp.org/news/why-i-left-gulp-and-grunt-for-npm-scripts-3d6853dd22b8/",
   },
+  {
+    Question:
+      "Which of the following type of Hackers look for vulnerable computers over the internet, exploit them and use them for their advantage?",
+    Answer: "Black Hat Hackers",
+    Distractor1: "Red Hat Hackers",
+    Distractor2: "White Hat Hackers",
+    Distractor3: "Grey Hat Hackers",
+    Explanation:
+      "Black Hat Hackers exploit vulnerable computers over the internet by installing backdoors, malwares, sending phishing emails and links , therby getting access to private and sensitive data",
+    Link: "https://www.freecodecamp.org/news/white-hat-black-hat-red-hat-hackers/",
+  },
+  {
+    Question:
+      "Which of the following type of Hackers protect civilians and organizations by finding and reporting vulnerabilites in the system?",
```

Comment:
It looks like we have a spelling error for vulnerabilities. 

```suggestion
      "Which of the following types of Hackers protect civilians and organizations by finding and reporting vulnerabilities in the system?",
```